### DISCUSSION TITLE
Train Arabic model

### DISCUSSION THREAD

**Discussion by Saifg1990** [yl4579/StyleTTS2#213] [General]
Hi, I want to train an Arabic model. Can you suggest good options? I want it to sound human like

**Comment by MohannadEhabBarakat** [comment]
Hi @Saifg1990, how did it go with this project? I'm working on the same project now

**Comment by MohannadEhabBarakat** [comment]
@Fadi987
Hi, I came across your amazing work in Arabic. Let's connect to work on projects together.
